src/main/kotlin/io/github/karmasmp/karma/player/visuals/PlayerVisuals.kt

```diff
@@ -26,14 +33,51 @@ import org.bukkit.util.Vector
 import java.time.Duration
 
 object PlayerVisuals {
+    private const val ADMIN_STAFF_MODE_IMAGE_SPACER = "<translate:space.-225>"
+    private const val ADMIN_STAFF_MODE_IMAGE_OFFHAND_SPACER = "<translate:space.-285>"
+    private const val ADMIN_IMAGE_SPACER = "<translate:space.-205>"
+    private const val ADMIN_IMAGE_OFFHAND_SPACER = "<translate:space.-265>"
+    private const val LIFE_IMAGE_SPACER = "<translate:space.-200>"
+    private const val LIFE_IMAGE_OFFHAND_SPACER = "<translate:space.-260>"
+
+    private const val GHOST_HEAD_TEXTURE = "eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvOTZjYzM3YjZjNWYxYWE1MWU5YzljOTYwMjNiNDJhM2NiOGI3MDg0N2MwNDE0OGUxMzViYjg4MTk3ZTBiMjRjYyJ9fX0="
+
+    fun actionBar(player: Player) {
+        object : BukkitRunnable() {
+            override fun run() {
+                if(player.isOnline) {
+                    if(player.isAdmin()) {
+                        if(player.isInStaffMode()) {
+                            player.sendActionBar(Formatting.allTags.deserialize("${if(player.inventory.itemInOffHand.type == Material.AIR) ADMIN_STAFF_MODE_IMAGE_SPACER else ADMIN_STAFF_MODE_IMAGE_OFFHAND_SPACER}<actionbar:admin><actionbar:staffmode>"))
+                        } else {
+                            player.sendActionBar(Formatting.allTags.deserialize("${if(player.inventory.itemInOffHand.type == Material.AIR) ADMIN_IMAGE_SPACER else ADMIN_IMAGE_OFFHAND_SPACER}<actionbar:admin>"))
+                        }
+                    } else {
+                        if(player.getKarmaLives() >= 3) {
```

Comment:
Kotlin when statement rather than if else chain for this simple check might be nicer, not sure if >=3 is possible with it though, if not then ignore

Reply:
Done